tests/schema.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
+    "root": {
+        "type": "object",
+        "properties": {
+            "readme": {"type": "string"},
+            "features": {"type": "object"},
+            "unprotectedTemporary": {
+                "type": "array",
+                "items": {"type": "object"}
+            }
+        }
+    },
+    "feature": {
```

Comment:
Is there a way we can describe unique keys for each feature?

Perhaps in their own schema file perhaps? It doesn't need to be in this PR just interested on your thoughts where it would go.

Reply:
Yeah we could define other schema files to match specific features.